internal/driver/glfw/window.go

```diff
@@ -78,7 +78,7 @@ type window struct {
 
 	xpos, ypos    int
 	width, height int
-	ignoreResize  bool
+	shouldExpand  bool
```

Comment:
Is there any possibility that a window shouldContract?

Reply:
No change in content will cause the window to contract, we just don't do that.
A window can be contracted in response to moving to a screen with lower DPI, but that is handled in the main window thread.